testbed/tf-script/xpanse-app.tf

```diff
@@ -0,0 +1,41 @@
+resource "clevercloud_addon" "xpanse_db" {
+  name                 = "db"
+  region               = "par"
+  plan                 = "dev"
+  third_party_provider = "mysql-addon"
+}
+
+resource "clevercloud_docker" "xpanse" {
+  name               = "xpanse"
+  region             = "par"
+  min_instance_count = 1
+  max_instance_count = 1
+  smallest_flavor    = "M"
+  biggest_flavor     = "M"
+
+  deployment {
+    repository = var.docker_repository_url
+    commit     = var.docker_commit
+  }
+
+  dockerfile = "testbed/clever-cloud/Dockerfile"
+
+  environment = {
+    CC_DOCKERFILE                       = "testbed/clever-cloud/Dockerfile"
+    SPRING_PROFILES_ACTIVE              = "dev,noauth,mysql,terra-boot,tofu-maker"
+    POLICY_MAN_ENDPOINT                 = "https://app-726ffe59-3913-4ce7-ac53-97d7b3751579.cleverapps.io"
```

Comment:
all end points must be automatically fetched similar to how you have done for mysql endpoint.